app/views/shelter_reviews/new.html.erb

```diff
@@ -1,13 +1,9 @@
 <section class = "info">
   <h2>Create Review:</h2>
   <% if !flash[:error].nil? %>
-  <div class = "error">
-    <% flash[:error].each do |msg| %>
-      <div>
-          <%= msg %>
-      </div>
-    <% end %>  
-  </div>
```

Comment:
Was able to get rid of this iteration logic in the view by calling `join` on the flash_message

Reply:
Nice!